config/checkstyle/checkstyle.xml

```diff
@@ -330,5 +330,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
         </module>
 
     </module>
-
+    <module name="SuppressionFilter">
```

Comment:
I prefer this style: https://checkstyle.sourceforge.io/config_filters.html#SuppressWarningsFilter